Tests/UnitTests/ess-list-encs.component.spec.ts

```diff
@@ -516,6 +516,22 @@ describe('EssListEncsComponent', () => {
     };
     expect(essInfoErrorMessageService.infoErrMessage).toStrictEqual(warnObj);
   });
+
+  test('getEstimatedTotalSize should return esimatedTotalSize when selectedEncList length is 0', () => {
+    component.selectedEncList = [];
+    component.getEstimatedTotalSize();
+    expect(component.selectedEncList.length).toBe(0);
+    expect(component.getEstimatedTotalSize()).toBe('0 MB');
+  });
+
+  test('getEstimatedTotalSize should return esimatedTotalSize when selectedEncList length is greater than 0', () => {
```

Comment:
Please correct the spelling mistake esimatedTotalSize to estimatedTotalSize

Reply:
Resolved